components/ide/jetbrains/image/status/main.go

```diff
@@ -322,6 +326,35 @@ func configureXmx(alias string) error {
 	return ioutil.WriteFile(launcherPath, []byte(newContent), 0)
 }
 
+func configureVMOptions(alias string) error {
```

Comment:
Supervisor can run IDE startup scripts several times. Status program should make sure that it produces the same result each time, i.e. does not prepend options over and over.

Reply:
Good catch, I will replace the prepend with replace & deduplicate